exercises/secret-handshake/src/main/kotlin/HandshakeCalculator.kt

```diff
@@ -0,0 +1,5 @@
+object HandshakeCalculator {
+    fun calculateHandshake(number: Int): List<Signal> {
+        return emptyList()
+    }
+}
```

Comment:
You might want to consider adding an empty newline at the EOF, I think this is common for most editors to insert, and will not pollute git changes when the final line is changed.